### PR TITLE
Merge main to develop after 2022.1 release

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: demos/face_detection/python/README.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 
-The script [face_detection.py](https://github.com/openvinotoolkit/model_server/blob/develop/demos/face_detection/python/face_detection.py) runs face detection inference requests for all the images
+The script [face_detection.py](https://github.com/openvinotoolkit/model_server/blob/releases/2022/1/demos/face_detection/python/face_detection.py) runs face detection inference requests for all the images
 saved in `input_images_dir` directory. 
 
 The script can adjust the input image size and change the batch size in the request. It demonstrates how to use
 the functionality of dynamic shape in OpenVINO Model Server and how to process the output from the server.
 
-The example relies on the model [face-detection-retail-0004](https://docs.openvinotoolkit.org/2021.4/omz_models_model_face_detection_retail_0004.html).
+The example relies on the model [face-detection-retail-0004](https://docs.openvinotoolkit.org/2022.1/omz_models_model_face_detection_retail_0004.html).
```
Comment (mzegla) [inline demos/face_detection/python/README.md:11]: We should also change this to .ai I guess

File: docs/developer_guide.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ Click the test that needs to be run:
 
 <details><summary>Run test inference</summary>
 
-1. Download an exemplary model [ResNet50-binary model](https://docs.openvinotoolkit.org/latest/omz_models_intel_resnet50_binary_0001_description_resnet50_binary_0001.html) :
+1. Download an exemplary model [ResNet50-binary model](https://docs.openvinotoolkit.org/2022.1/omz_models_intel_resnet50_binary_0001_description_resnet50_binary_0001.html) :
```
Comment (mzegla) [inline docs/developer_guide.md:102]: We should probably change to .ai I guess

File: docs/dynamic_shape_custom_node.md [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
-The node below is provided as a demonstration. See instructions for how to build and use the custom node: [Image Transformation](https://github.com/openvinotoolkit/model_server/tree/develop/src/custom_nodes/image_transformation).
+The node below is provided as a demonstration. See instructions for how to build and use the custom node: [Image Transformation](https://github.com/openvinotoolkit/model_server/tree/releases/2022/1/src/custom_nodes/image_transformation).
 
 
 To run inference with this setup, we will use the following:
 
-- Example client in Python [face_detection.py](https://github.com/openvinotoolkit/model_server/blob/develop/demos/face_detection/python/face_detection.py) that can be used to request inference on with the desired input shape.
+- Example client in Python [face_detection.py](https://github.com/openvinotoolkit/model_server/blob/releases/2022/1/demos/face_detection/python/face_detection.py) that can be used to request inference on with the desired input shape.
 
 - An example [face_detection_retail_0004](https://docs.openvinotoolkit.org/2021.4/omz_models_model_face_detection_retail_0004.html) model.
```
Comment (mzegla) [inline docs/dynamic_shape_custom_node.md:13]: We should probably change to .ai + new tag I guess

File: docs/dynamic_shape_dynamic_model.md [hunk 4]
```diff
@@ -8,13 +8,13 @@ Enable dynamic shape by setting the `shape` parameter to range or undefined:
...
+on [Shape Inference Document](https://docs.openvino.ai/2022.1/openvino_docs_IE_DG_ShapeInference.html).
 
 Another option to use dynamic shape feature is to export the model with dynamic dimension using Model Optimizer. OpenVINO Model Server will inherit the dynamic shape and no additional settings are needed.
 
 To the demonstrate dynamic dimensions, take advantage of:
 
-- Example client in Python [face_detection.py](https://github.com/openvinotoolkit/model_server/blob/develop/demos/face_detection/python/face_detection.py) that can be used to request inference with the desired input shape.
+- Example client in Python [face_detection.py](https://github.com/openvinotoolkit/model_server/blob/releases/2022/1/demos/face_detection/python/face_detection.py) that can be used to request inference with the desired input shape.
 
 - An example [face_detection_retail_0004](https://docs.openvinotoolkit.org/2021.4/omz_models_model_face_detection_retail_0004.html) model.
```
Comment (mzegla) [inline docs/dynamic_shape_dynamic_model.md:19]: We should probably change to .ai + new tag I guess

**Review by mzegla — APPROVED** [review]

**Review by dtrawins — APPROVED** [review]

**Pull request merged**